docs/projects.md

```diff
@@ -69,6 +69,11 @@
         "url": "/hardware",
         "imageUrl": "/static/hardware/ghiarcade.jpg"
     },
+    {
+        "name": "How to Make a Game Videos",
```

Comment:
can we have a name more specific? we have tons of games here. what about the cat platformer game?

Reply:
Well it's got the bunny hop game in there too and I'm planning to add a bunch more videos about making specific kinds of games

Reply:
I guess I'll take silence as assent?